docs/content/apis/dmf_api.md

```diff
@@ -364,6 +364,8 @@ type=EVENT  <br /> tenant=tenant123 <br /> thingId=abc  <br /> topic=DOWNLOAD\_A
 
 If `multi.assignments.enabled` is enabled, this message is sent instead of DOWNLOAD_AND_INSTALL, DOWNLOAD, or CANCEL_DOWNLOAD
  by hawkBit to initialize update, download, or cancel task(s).
+ 
+ With weight, one can set the priority to the action. Higher the weight, more is the priority of an action.
```

Comment:
The higher the weight, the higher the priority of an action.

Reply:
corrected the sentence